doc/sphinx-guides/source/container/app-image.rst

```diff
@@ -0,0 +1,220 @@
+  | *Note:* default is ``gdcc/base:${base.image.tag}``. See also :doc:`base-image` for more details on it.
+
+Automated Builds & Publishing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+See note above at "Supported Image Tags".
+
+.. _app-multiarch:
+
+Processor Architecture and Multiarch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This image is created as a "multi-arch image", supporting the most common architectures Dataverse usually runs on:
+AMD64 (Windows/Linux/...) and ARM64 (Apple M1/M2), by using `Maven Docker Plugin's BuildX mode <https://dmp.fabric8.io/#build-buildx>`_.
+
+Building the image via ``mvn -Pct package`` or ``mvn -Pct install`` as above will only build for the architecture of
+the Docker machine's CPU.
+
+Only ``mvn -Pct clean deploy -Ddocker.platforms=linux/amd64,linux/arm64`` will trigger building on all enabled architectures.
+Yet, to enable building with non-native code on your build machine, you will need to setup a cross-platform builder.
+
+On Linux, you should install `qemu-user-static <https://github.com/multiarch/qemu-user-static>`__ (preferably via
+your package management) on the host and run ``docker run --rm --privileged multiarch/qemu-user-static --reset -p yes``
+to enable that builder. The Docker plugin will setup everything else for you.
+
+
+
+.. _app-tunables:
+
+Tunables
+++++++++
+
+The :doc:`base-image` provides a long list of possible options to tune many aspects of the application server, and,
+as the application image builds upon it, :ref:`Base Image Tunables <base-tunables>` apply to it as well.
+
+In addition, the application image provides the following tunables:
+
+.. list-table::
+    :align: left
+    :width: 100
+    :widths: 10 10 10 50
+    :header-rows: 1
+
+    * - Env. variable
+      - Default
+      - Type
+      - Description
+    * - ``MP_CONFIG_PROFILE``
+      - ``ct``
+      - String
+      - Set to switch the activated *MicroProfile Config Profile*. Note that certain defaults will not apply any longer.
+        See :ref:`:ApplicationServerSettings` for details.
+    * - ``dataverse_*`` and ``doi_*``
+      - \-
+      - String
+      - Configure any :ref:`jvm-options` not yet *MicroProfile Config* enabled with this magic trick.
+
+        1. Simply pick a JVM option from the list and replace any ``.`` with ``_``.
+        2. Replace any ``-`` in the option name with ``__``.
+    * - ``DATAVERSE_MAIL_HOST``
+      - ``smtp``
+      - String
+      - A hostname (w/o port!) where to reach a Mail MTA on port 25.
+    * - ``DATAVERSE_MAIL_USER``
+      - ``dataversenotify``
+      - String
+      - A username to use with the Mail MTA
+    * - ``DATAVERSE_MAIL_FROM``
+      - ``dataverse@localhost``
+      - Mail address
+      - The "From" field for all outbound mail. Make sure to set :ref:`systemEmail` to the same value or no mail will
+        be sent.
+
+
+Note that the script ``init_2_configure.sh`` will apply a few very important defaults to enable quick usage
```

Comment:
I'm not sure is this section, with implementation details, contains necessary information for someone using the image.

Reply:
Well this becomes very important if you configure the app to use some S3 storage... By switching the default storage to something other than "local", the local file storage will vanish.

Reply:
Right now the target user is a developer, someone hacking on the Dataverse software itself. I think these developers have enough information, right? Can we pick up this thread in a future pull requests when there is a different, non-developer audience?